ui/page/send/send_destination.go

```diff
@@ -95,7 +101,7 @@ func (dst *destination) validateDestinationAddress() (string, error) {
 		return address, fmt.Errorf(values.String(values.StrDestinationMissing))
 	}
 
-	if dst.WL.SelectedWallet.Wallet.IsAddressValid(address) {
+	if dst.selectedWallet.IsAddressValid(address) {
```

Comment:
This should use `dst.destinationWalletSelector.SelectedWallet().IsAddressValid(...)`. That is more correct.